### PR TITLE
Add search growth EM job description

### PR DESCRIPTION
**Pull request by lguychard**
The search team will have 6+ engineers soon, and we need to start thinking about splitting it. This PR adds a job description for an engineering manager for the future Search growth team.

### CODE REVIEW

File: handbook/engineering/hiring/engineering-manager-search-growth.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+1. Intro calls
+   - 45m [Nick Snyder](../../../company/team/index.md#nick-snyder-he-him) (VP Engineering)
+   - 30m [Loïc Guychard](../../../company/team/index.md#loic-guychard) (Engineering Manager, Search)
+1. Interviews.
+   - 1h [Engineering leadership](engineering-leadership.md).
+     - The interviewers will be [Nick Snyder](../../../company/team/index.md#nick-snyder-he-him) and another Engineering Manager except Loïc.
+   - 1h [Technical experience](https://github.com/sourcegraph/interviews/blob/master/engineering/technical-experience.md) - We ask you about your past work and accomplishments.
+     - The interviewers will be two of the following:
+       - [Juliana Peña](../../../company/team/index.md#keegan-pena-she-her)
```
Comment (limitedmage) [inline handbook/engineering/hiring/engineering-manager-search-growth.md:51]:
```suggestion
       - [Juliana Peña](../../../company/team/index.md##juliana-peña-she-her)
```

Comment (lguychard) [reply handbook/engineering/hiring/engineering-manager-search-growth.md:51]: omg... sorry 🤦 🤣  I am _merging search team members_

File: handbook/engineering/search/index.md [hunk 2]
```diff
@@ -84,17 +84,22 @@ Loïc will be at management capacity when we reach 7-8 engineers on the team, bu
...
 
 The Search growth team will focus on initiatives aiming at growing usage or adoption of our search features, such as onboarding efforts or features unlocking new use cases.
 
-Examples:
+We are looking to hire an [engineering manager](../hiring/engineering-manager-search-growth.md) and a [Product Manager](../../product/roles/product_manager.md) for this team.
+
+Example projects:
 
-- Implement code monitoring.
+- Version contexts
```
Comment (rvantonder) [inline handbook/engineering/search/index.md:103]:
```suggestion
- Version contexts.
```

File: handbook/engineering/hiring/engineering-manager-search-growth.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+As our [search team plans to split](../search/index.md#growth-plan), we are looking for an engineering manager to build and lead our [Search growth](../search/index.md#search-growth) team.
+
+The team will own unlocking new use cases of Sourcegraph search and improving adoption and retention of Sourcegraph search features for Cloud and Enterprise users alike, through projecs such as:
+
+- Adding [code monitoring](https://docs.google.com/document/d/1_R5DgpUkxyZilsJ9vBQm5cvRPT2udc3tZIPg2q3cnZU/edit) capabilities to Sourcegraph
+- Improving user onboarding to the Sourcegraph query language
+- Making Sourcegraph search aware of [contexts](https://docs.google.com/document/d/1mlxy7Fy19Q2yua7Fjg0xCda1c07f9RoR8rXxU22Ni60/edit) that help users search through the code they care about
+
+While some engineers will join the Search growth team immediately following the split, the team will likely need to grow to achieve its goals, and you own defining hiring needs and execuing on them.
```
Comment (rewislam) [inline handbook/engineering/hiring/engineering-manager-search-growth.md:11]:
```suggestion
While some engineers will join the Search growth team immediately following the split, the team will likely need to grow to achieve its goals, and you own defining hiring needs and executing on them.
```

**Comment by lguychard** [comment]
Holding off on adding this for now: [context](https://sourcegraph.slack.com/archives/CQ5DH0NEB/p1607982659096000).

**Pull request closed without merging**